mutation-testing/filter-pr/action.yml

```diff
@@ -0,0 +1,98 @@
+name: Filter PR Mutants
+
+runs:
+  using: 'composite'
+
+  steps:
+    # Cleanup Runner
+    - name: Cleanup Runner action repo
+      id: runner_cleanup
+      uses: stacks-network/actions/cleanup/disk@main
+
+    # Checkout the stacks-core code
+    - name: Checkout the stacks core repo
+      id: git_checkout_stacks_core
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
`fetch-depth: 0` fetches entire history. Can we optimize by only fetching the branches (or commits) we are comparing?